Makefile

```diff
@@ -1,6 +1,8 @@
 LIBDIR := lib
 USE_XSLT := true
 DISABLE_RIBBON := true
+GHPAGES_EXTRA = $(foreach ext,.html .txt,$(addsuffix $(ext),$(foreach draft,$(drafts),$(shell echo $(draft) | sed -e 's/draft-ietf-httpbis-//'))))
```

Comment:
Use `:=` or the expansion of GHPAGES_EXTRA will run the loop every time that `$(GHPAGES_EXTRA)` appears.

Reply:
That's intentional, actually.  $(drafts) is undefined until after the import, at which point it's too late.  By delaying expansion, it *is* defined by the time GHPAGES_EXTRA is used.

Reply:
Ugh, oh well.  There should only be two expansions, so that isn't so bad.